tests/Sapi/CliServer/CliServer.php

```diff
@@ -93,6 +93,24 @@ public function start()
 
         $this->process = new Process($processCmd);
         $this->process->start();
+
+        if (!$this->waitUntilServerRunning()) {
+            error_log("[cli-server] Server never came up...");
+        }
+    }
+
+    public function waitUntilServerRunning()
```

Comment:
This is here to abort the following error `Failed to connect to localhost port 9996: Connection refused, error code 7`